typespec-tests/src/main/java/com/cadl/flatten/implementation/JsonMergePatchHelper.java

```diff
@@ -32,6 +26,12 @@ public static UpdatePatchRequestAccessor getUpdatePatchRequestAccessor() {
         return updatePatchRequestAccessor;
     }
 
+    private static TodoItemPatchAccessor todoItemPatchAccessor;
```

Comment:
double confirm: This change is just switch order?

Reply:
Most of this is reordering so the field, interface, and static methods for a type are together instead of the normal layout where all field declarations are together, then interfaces, and finally static methods. I made this change as if you're going to the helper class you are likely caring about a specific access helper and this makes it easier to find all the information for it.